src/app/component/auth/components/sign-up/sign-up.component.scss

```diff
@@ -6,220 +6,186 @@ $title-font: 'Montserrat', sans-serif;
 $main-font: 'Open Sans', sans-serif;
 $sign-up-google-font: 'Roboto';
 
-.main {
-  width: 99vw;
-  overflow-y: auto;
-  overflow-x: hidden;
-  max-height: calc(100vh - -64px);
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  z-index: 2000;
-}
+.main-content-container {
+  margin: 0 55px;
+  width: 370px;
+
+  .title-text {
+    margin: 0 0 8px;
+    font-family: $title-font;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 40px;
+    color: #2b292d;
+  }
 
-.main-container {
-  display: flex;
-  max-width: 960px;
-  font-family: $main-font;
-  background-color: white;
+  .subtitle-text {
+    font-size: 16px;
+    line-height: 16px;
+    color: #494a49;
+  }
 
-  .close-btn {
-    margin: 24px 24px 0 20px;
+  .form-content-container {
     display: flex;
-    justify-content: flex-end;
+    flex-direction: column;
+    margin: 19px 0 40px;
 
-    .cross-img {
-      width: 12px;
-      height: 12px;
+    .content-label {
+      margin: 21px 0 4px;
+      font-size: 12px;
+      line-height: 14px;
+      color: #494a49;
     }
-  }
-
-  .main-image {
-    border-radius: 4px 0 0 4px;
-  }
 
-  .main-content-container {
-    margin: 0 55px;
-    width: 370px;
+    .under-error {
+      margin: 9px 0 4px;
```

Comment:
why 9 and 4 ?

Reply:
according to design
![2020-10-26 (2)](https://user-images.githubusercontent.com/62401277/97202642-8d7cf800-17bc-11eb-8295-870551aa35c8.png)